### PR TITLE
Update cli text. Addresses #1173

### PR DESCRIPTION
**Pull request by harsimranmaan**

### CODE REVIEW

File: buffalo/cmd/build.go [hunk 1]
```diff
@@ -52,7 +52,7 @@ var xbuildCmd = &cobra.Command{
 			return errors.WithStack(err)
 		}
 
-		logrus.Infof("\nYour application was successfully built at %s\n", b.AbsoluteBinaryPath())
+		logrus.Infof("\nThe pplication was successfully built at %s\n", b.AbsoluteBinaryPath())
```
Comment (mclark4386) [inline buffalo/cmd/build.go]: *a*pplication

Comment (harsimranmaan) [reply buffalo/cmd/build.go]: Fixed. Thanks

**Comment by mclark4386** [comment]
Also all new PRs like this should be against `development` not `master`. Adjusted that for you, but there seems to be a conflict.

**Comment by harsimranmaan** [comment]
Yeah. I saw that after I fixed the link to Contributing.md in #1387 . Rebased.

**Comment by markbates** [comment]
Thanks!

**Pull request merged**